web2/sdk/contracts/flexi-swap-contract.ts

```diff
@@ -0,0 +1,68 @@
+import { write } from "./../utils";
+import { prepareWriteContract } from "@wagmi/core";
+import { FLEXISWAP_ABI } from "../constants";
+import { ItemInfo } from "../types";
+
+export class FlexiSwapContract {
```

Comment:
Lets call this FlexiSwapAdapter. Don't forget to rename file as well and code occurrences